python/doc/examples/calibration/least_squares_and_gaussian_calibration/plot_calibration_chaboche.py

```diff
@@ -436,19 +642,32 @@
 
 # %%
 graph = calibrationResult.drawResiduals()
-view = viewer.View(graph)
+view = viewer.View(
+    graph,
+    figure_kw={"figsize": (6.0, 4.0)},
+    legend_kw={"bbox_to_anchor": (1.0, 1.0), "loc": "upper left"},
+)
+plt.subplots_adjust(right=0.6)
 
 # %%
-# The analysis is very similar to the linear calibration.
+# The analysis is very similar to the linear Gaussian calibration.
 
 # %%
 graph = calibrationResult.drawParameterDistributions()
-view = viewer.View(graph)
+view = viewer.View(
+    graph,
+    figure_kw={"figsize": (10.0, 4.0)},
+    legend_kw={"bbox_to_anchor": (1.0, 1.0), "loc": "upper left"},
+)
+plt.subplots_adjust(right=0.8)
 
 # %%
 graph = calibrationResult.drawResidualsNormalPlot()
 view = viewer.View(graph)
 
 plt.show()
 # %%
-# We see that the prior and posterior distribution for the :math:`\gamma` parameter are close to each other, but not superimposed: the observations significantly brought information to the variable :math:`\gamma` during the calibration.
+# We see that the prior and posterior distribution for the :math:`\gamma`
+# parameter are close to each other, but not superimposed: the observations
+# significantly brought information to the variable :math:`\gamma` during
+# the calibration.
```

Comment:
This paragraph is misplaced, I think. It is not related to the residual vs Gaussian hypothesis plot.

Reply:
Indeed. I misread the document, sorry!

Reply:
Ok.